qrenderdoc/CMakeLists.txt

```diff
@@ -185,8 +187,15 @@ file(WRITE
      "LIBS+=${PYTHON_LINK}\n"
      "\n"
      "RENDERDOC_VERSION=${RENDERDOC_VERSION}\n"
-     "\n"
-     "OSX_ICONFILE=${CMAKE_CURRENT_BINARY_DIR}/RenderDoc.icns\n")
+     "\n")
+
+if(APPLE)
```

Comment:
I don't understand why this code has been moved out into this `if()`? `QMAKE_RPATHDIR` is the only property that would be relevant for other platforms and it's not set to anything except on apple.

Reply:
No particular reason tidying up the APPLE specific changes to be behind if (APPLE).
When QMAKE_RPATHDIR was added did not notice the pattern to use if (APPLE) and append to the qmake config file. 

Happy to revert this change

Reply:
Dropped the commit from history